build/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM alpine:3.8
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+ENV OPERATOR_PATH=/go/src/github.com/openshift/pagerduty-operator
+COPY . ${OPERATOR_PATH}
+WORKDIR ${OPERATOR_PATH}
+RUN make gobuild
 
-RUN apk --no-cache add ca-certificates
```

Comment:
if the pagerduty operator connects to pagerduty or other external URLs via https it might need the ca-certificates package for the Root CAs.

Reply:
If that's the case we should install it on the 2nd stage of the Dockerfile

Reply:
We're not supposed to use alpine though.

Reply:
We could install it with yum, like `yum install -y ca-certificates && yum clean all`